utils/hexedit/Makefile

```
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2007-2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hexedit
+PKG_VERSION:=1.6
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/pixel/hexedit.git
+PKG_SOURCE_VERSION:=eab92dcaa34b66bc5182772afc9fda4ac8a27597
+PKG_MIRROR_HASH:=138f8aff0466a33f33be43e25e7a47ba48216c0b2d41cbb6cc23ca848bd8a564
+
+PKG_LICENSE:=GPL-2.0-only
```

```suggestion
PKG_LICENSE:=GPL-2.0-or-later
```

Based on [code comment](https://github.com/pixel/hexedit/blob/eab92dcaa34b66bc5182772afc9fda4ac8a27597/hexedit.c#L6-L7).